fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/expressions/Field.java

```diff
@@ -57,13 +57,25 @@ public QueryComponent matches(@Nonnull QueryComponent child) {
     /**
      * If the associated field is a repeated one, this allows you to match against one of those values, the record will
```

Comment:
This wasn't changed *per se*, but it is also a comma splice; it would be great if this could be changed to match the new description below.